app/client/sagas/authSagas.js

```diff
@@ -464,8 +464,7 @@ function* watchInviteUserRequest() {
 function* validateTFA({ payload }) {
   try {
     const validateTFAresponse = yield call(ValidateTFAAPI, payload);
-
-    yield call(updateOrganisationStateFromLoginData, activated_account);
```

Comment:
`activated_account ` was undefined, causing the saga to throw an error. We should implement basic frontend tests for all core paths